python/lsst/afw/math/boundedField.cc

```diff
@@ -69,11 +69,14 @@ PYBIND11_PLUGIN(_boundedField) {
     clsBoundedField.def("__truediv__", &BoundedField::operator/);
 
     /* Members */
-    clsBoundedField.def("evaluate", (double (BoundedField::*)(double, double) const)
-        &BoundedField::evaluate);
-    clsBoundedField.def("evaluate", (ndarray::Array<double,1,1>
-        (BoundedField::*)(ndarray::Array<double const,1> const &,
-                          ndarray::Array<double const,1> const &) const) &BoundedField::evaluate);
+    clsBoundedField.def("evaluate", (double (BoundedField::*)
+        (double, double) const) &BoundedField::evaluate);
+    clsBoundedField.def("evaluate", (ndarray::Array<double,1,1> (BoundedField::*)
+        (ndarray::Array<double const,1> const &,
+         ndarray::Array<double const,1> const &) const) &BoundedField::evaluate);
+    clsBoundedField.def("evaluate", (double (BoundedField::*)
+        (lsst::afw::geom::Point2D const &) const) &BoundedField::evaluate);
```

Comment:
This was a mistake and shouldn't have been committed. Re: using `clang_format` in general, I agree with @pschella that it's probably better to wait until the new rules are finalized.